### PR TITLE
Add Swift Package Manager section in `Installation` guide in `README.md` file.

### PR DESCRIPTION
**Pull request by auhustsinovich**
## What were done

- Provided documentation about integrating `Collections` framework via Swift Package Manager.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,23 +28,45 @@ This project uses [Semantic Versioning 2.0.0](https://semver.org/).
...
 
-  To build the binary `.xcframework`, run the `generate-xcframework.sh` script in a [scripts](./scripts/) folder:
-  ```bash
-  chmod u+x ./scripts/generate-xcframework.sh
-  ./scripts/generate-xcframework.sh
-  ```
-  or follow [instructions](https://developer.apple.com/documentation/xcode/creating-a-multi-platform-binary-framework-bundle#Create-archives-for-frameworks-or-libraries) provided by   Apple.
+1. Open your Xcode project.
+1. Select "File" → "Add Packages..."
+1. In the search bar, enter the URL of the `Collections` repository: `https://github.com/auhustsinovich/Collections/`
```
[the remark this answers is not in the document]
Comment (auhustsinovich) [reply README.md:41]: fixed.

[the remark this answers is not in the document]
Comment (auhustsinovich) [reply README.md:41]: fixed.

File: README.md [hunk 2]
```diff
@@ -28,23 +28,45 @@ This project uses [Semantic Versioning 2.0.0](https://semver.org/).
...
+
+```swift
+dependencies: [
+    .package(url: "https://github.com/auhustsinovich/Collections", from: "<version>")
+]
+```
+
+### Download the binary
+
+1. Download the release of the `Collections.xcframework` from the GitHub releases page.
```
Comment (AlexLevshenkov) [inline README.md:56]: Please add a link to `Collections` releases page

File: README.md [hunk 3]
```diff
@@ -28,23 +28,45 @@ This project uses [Semantic Versioning 2.0.0](https://semver.org/).
...
+1. Add `Collections.xcframework` to your project (select the `Do Not Embed` option).
```
Comment (AlexLevshenkov) [inline README.md:57]:
```suggestion
1. Add `Collections.xcframework` to your project (The `Do Not Embed` is the recommended option).
```

File: README.md [hunk 4]
```diff
@@ -28,23 +28,45 @@ This project uses [Semantic Versioning 2.0.0](https://semver.org/).
...
+#### Adding `Collections` as a dependency in your Xcode project
 
-  To build the binary `.xcframework`, run the `generate-xcframework.sh` script in a [scripts](./scripts/) folder:
-  ```bash
-  chmod u+x ./scripts/generate-xcframework.sh
-  ./scripts/generate-xcframework.sh
-  ```
-  or follow [instructions](https://developer.apple.com/documentation/xcode/creating-a-multi-platform-binary-framework-bundle#Create-archives-for-frameworks-or-libraries) provided by   Apple.
+1. Open your Xcode project.
+1. Select "File" → "Add Packages...".
```
[the remark this answers is not in the document]
Comment (auhustsinovich) [reply README.md:40]: thanks for noting, fixed!

File: README.md [hunk 5]
```diff
@@ -28,23 +28,45 @@ This project uses [Semantic Versioning 2.0.0](https://semver.org/).
...
+1. In the search bar enter the URL of the `Collections` repository: `https://github.com/auhustsinovich/Collections/`.
```
[the remark this answers is not in the document]
Comment (auhustsinovich) [reply README.md:41]: fixed!

File: README.md (hunk 3 above)
Comment (auhustsinovich) [reply README.md:57]: fixed!

File: README.md (hunk 2 above)
Comment (auhustsinovich) [reply README.md]: Added a link, fixed.

**Review by AlexLevshenkov — APPROVED** [review]

**Pull request merged**